src/libm-tester/CMakeLists.txt

```diff
@@ -24,21 +25,28 @@ set(IUT_LIST ${TARGET_IUT})
 
 set(IUT_SRC iutsimd.c iutsimdmain.c testerutil)
 
+if (CMAKE_SYSTEM_NAME MATCHES "CYGWIN")
```

Comment:
Why do you need to skip these tests? It seems to me that you are building them, even on cygwin, (see line 35), but then you decide not to run them. Aren't the iuts just supposed to bail out if the platform does not support the vector architecture? Or am I missing something?

Having said that, I would prefer not to mix different aspects of the build system in the same pull request. I would prefer you to move the cygwin specific bits from here to a new pull request and limit this one to SLEEF_STATIC_LIBS.